src/components/SeedsSelectedList/index.js

```diff
@@ -43,14 +76,27 @@ const SeedsSelectedList = ({ list }) => {
     >
       {[...list].reverse().map((s, i) => (
         <Box minWidth={matchesMd ? '120px' : ''} key={i}>
+
           <Card
             sx={{
               backgroundColor: 'transparent',
               boxShadow: 'none',
               cursor: 'pointer',
             }}
           >
-            <CardActionArea onClick={() => selectSpecies(s.label)}>
+            <CardActionArea onClick={() => {
```

Comment:
rather than doing a list of items in the on click.  lets have the onClick call a function that then does all of these things.  
could be:
`const cardActionClick = () =>{
        selectSpecies(s.label); 
        dispatch(removeSeedRedux(s.label));
        dispatch(removeOptionRedux(s.label));
}`

and this line could be 
`onClick={() => cardActionClick()}`

Reply:
I also noticed that this function is conflicted with a previous feature, which is `selectSpecies()`, this is a feature on further pages. It will set clicked seedicon to redux, and it will open accordion based on clicked seed. You can switch to dev branch to check this feature.

One some possible solutions I can think is to keep the delete function on this page, and switch it to `selectSpecies()` in the further page(maybe also hide the x icon), since I don't think we should delete the species after selection.